R/cnd-abort.R

```diff
@@ -220,6 +236,36 @@ signal_abort <- function(cnd) {
 
   stop(fallback)
 }
+cnd_unhandled_message <- function(cnd) {
+  paste_line(
+    conditionMessage(cnd),
+    format_onerror_backtrace(cnd)
+  )
+}
+
+on_load({
+  s3_register("knitr::sew", "rlang_error", function(x, options, ...) {
+    # Simulate interactive session to prevent full backtrace from
+    # being included in error message
+    local_interactive()
+
+    # Save the unhandled error for `rlang::last_error()`.
+    last_error_env$cnd <- x
+
+    # By default, we display no reminder or backtrace for errors
```

Comment:
What about errors not captured by knitr?

Reply:
As before they fall through the normal case and display a reminder in interactive sessions, or a backtrace in non-interactive sessions:

```r
Quitting from lines 34-36 (test-trace-full.Rmd) 
Error: foo
Backtrace:
    ▆
 1. └─global::f()
 2.   └─global::g()
 3.     └─global::h()
```